trestle/common/const.py

```diff
@@ -508,6 +508,18 @@
   #
 """
 
+YAML_RULE_PARAM_VALUES_COMPONENT_COMMENT = """  # You may set new values for rule parameters by adding
+  #
+  # component-values:
+  #   - value 1
+  #   - value 2
+  #
+  # below a section of values:
```

Comment:
How does the values relate to the param-id ? In other words, how can I set a specific set-parameter in component-definition?

Reply:
I think it will be clear to users that each "values:" list is bound to a single parameter id, and the new "component-values:" goes directly below it.  If you see the full markdown header with the comment it should make sense.

Reply:
I see, that make sense then. Setting multiple values for the different parameters could be a good test case as well